pp_sys.c

```diff
@@ -3972,7 +3972,7 @@ PP_wrapped(pp_chdir, MAXARG, 0)
 #endif
     }
     else 
-        PUSHi( PerlDir_chdir(tmps) >= 0 );
+        PUSHs(boolSV( PerlDir_chdir(tmps) >= 0 ));
```

Comment:
This feels like we should add a `PUSHb` macro.

Reply:
Perhaps, but aren't the `PUSH*` macros going to go away once we have a fully refcounted stack?

(Anyway, this is definitely out-of-scope for this PR.)